js/web/test/test-runner.ts

```diff
@@ -553,6 +565,50 @@ function createGpuTensorForOutput(type: ort.Tensor.Type, dims: readonly number[]
   });
 }
 
+const getContext = (() => {
+  let context: MLContext|undefined;
+
+  return async(): Promise<MLContext> => {
+    if (!context) {
+      context = await navigator.ml.createContext();
```

Comment:
Lack of `deviceType` info to create the `MLContext`, even it's not the same context as the one created by WebNN EP. `MLBuffer` requires shared `MLContext` for IO binding. For `ml-tensor`, we should create the `MLContext` from `test-runner-cli.ts` then pass to this file.

Reply:
Instead of creating the context in `test-runner-cli.ts`, I ended up adding create code to `create`. Not sure which is better, but `test-runner.ts` was already checking for the `webnn` backend's `executionProviderConfig`.

We were also missing `webnn` under `"Ignoring IO Binding Mode"` check in `test-runner-cli.ts`.